source/common/common/logger.h

```diff
@@ -76,7 +76,8 @@ namespace Logger {
   FUNCTION(tracing)                                                                                \
   FUNCTION(upstream)                                                                               \
   FUNCTION(udp)                                                                                    \
-  FUNCTION(wasm)
+  FUNCTION(wasm)                                                                                   \
+  FUNCTION(sip)
```

Comment:
We don't have a good solution for this right now but I would prefer to not add a new logging category for a contrib extension. Please remove this and use something generic for now like filter, udp, etc.

Reply:
removed.